searx/data/wikidata_units.json

```diff
@@ -281,7 +283,6 @@
     "Q15551713": "Sh",
     "Q1569733": "St",
     "Q15784325": "apc",
-    "Q160680": "Br",
```

Comment:
@dalf: do you know why Br (Belarusian ruble / from 2000–2016 ) is deleted? .. AFAIK it still exists in the Database https://www.wikidata.org/wiki/Q160680

Reply:
The request selects only item which are instances of (wdt:P31/wdt:P279) [unit of measurement, Q47574](https://www.wikidata.org/wiki/Q47574) : 
https://github.com/searxng/searxng/blob/62ea3c7074802abd3c312642feb832aae494c3d2/searx_extra/update/update_wikidata_units.py#L24-L35
* you can run the query only at this URL: https://query.wikidata.org
* `?item wdt:P31/wdt:P279 A` select instances of A or any subclass of A. For example, in [Q47574](https://www.wikidata.org/wiki/Q47574), you see that "unit of measurement" is a subclass ([P279](https://www.wikidata.org/wiki/Property:P279)) of [measure](https://www.wikidata.org/wiki/Q39875001).
* `?item wdt:P31/wdt:P279* A` is the same thing with any depth (you can see that the request returns more results). Actually, the request in update_wikidata_units.py should use `*`.

Q160680 is not a unit of measurement: in wikidata, money are not unit of measurement.
For example [Euro, Q4916](https://www.wikidata.org/wiki/Q4916) is instance of currency and decimal currency.
So my guess is Q160680 had the wrong "instance of" values for a time.

Reply:
Ah, .. thanks for the explanation .. I should bookmark your comment .. extremely instructive, your explanation 👍🏻